vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/LiveLocationShareMessageItemFactory.kt

```diff
@@ -75,7 +75,8 @@ class LiveLocationShareMessageItemFactory @Inject constructor(
         val height = dimensionConverter.dpToPx(MessageItemFactory.MESSAGE_LOCATION_ITEM_HEIGHT_IN_DP)
 
         return MessageLiveLocationInactiveItem_()
-                .attributes(attributes)
+                // disable the click on this state item
+                .attributes(attributes.copy(itemClickListener = null))
```

Comment:
Maybe we can avoid to add the clickListener before or add a new boolean to disable instead?

Reply:
I am not sure I can better and more simple for that. Deciding whether the item should be clickable depends on the status of the live which is computed inside this factory using some fields of the summary data. Unless you have a solution in mind?